packages/dashboard/src/components/Card/animations/DonutChart.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+import { getStyles, IDonutChartStyles } from './DonutChart.style';
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+
+export interface IDonutChartProps {}
+
+export class DonutChart extends React.Component<IDonutChartProps> {
+  constructor(props: IDonutChartProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    const getClassNames = classNamesFunction<IDonutChartProps, IDonutChartStyles>();
```

Comment:
This can be declared before the `DonutChart` class definition, to avoid redefining it every time the `render` method is called.